Update TabPanel readme

kjellr opened the pull request:
Adding documentation to describe the use and functionality of the TabPanel component.

Thanks @melchoyce and @alexislloyd for drafting this.

kjellr committed:
Update TabPanel readme

Adding documentation to describe the use and functionality of the TabPanel component.

Thanks @melchoyce and @alexislloyd for drafting this.

melchoyce commented on packages/components/src/tab-panel/README.md:
```diff
@@ -1,12 +1,59 @@
+
+![The “Document” tab selected in the sidebar TabPanel.](https://wordpress.org/gutenberg/files/2019/01/s_E36D9C9B8FFA15A1A8CE224E422535A12B016F88884089575F9998E52016A49F_1541785098230_TabPanel.png)
+
+
```
Extra space

kjellr replied:
Thanks! Fixed in 3e6a855

melchoyce commented on packages/components/src/tab-panel/README.md:
```diff
@@ -1,12 +1,59 @@
+![The “Document” tab selected in the sidebar TabPanel.](https://wordpress.org/gutenberg/files/2019/01/s_E36D9C9B8FFA15A1A8CE224E422535A12B016F88884089575F9998E52016A49F_1541785098230_TabPanel.png)
+
+
+# Table of contents
```
Heading hierarchy needs updating — everything except the title needs to get pushed down a level (so this would be an h2, etc.)

kjellr replied:
Should be fixed in 10f7c21

melchoyce reviewed (changes requested):
With the one space and the heading issues resolved, this should be good to go.

kjellr committed:
Remove extra space.

kjellr committed:
Decrease headling levels.

The page title should be the only H1, so everything else needs to get pushed down one level.

kjellr commented on packages/components/src/tab-panel/README.md:
```diff
@@ -38,35 +84,35 @@ const MyTabPanel = () => (
 The component accepts the following props:
 
-### className
+#### className
```
@melchoyce — should these be H4s (like the are currently), or should they just be bold? I see you're using bold in #13320.

melchoyce replied:
I'm not sure — what did we use in the other doc update PRs? I can change mine to match if we need.

kjellr replied:
Looks like we've been inconsistent: 

H4's: 
https://github.com/WordPress/gutenberg/pull/11226
https://github.com/WordPress/gutenberg/pull/12485

Bold Text: 
https://github.com/WordPress/gutenberg/pull/11872

melchoyce replied:
Let's go with H4s, looks like it's just me being inconsistent.

kjellr commented:
This should be good to go after 3e6a855 and 10f7c21. @melchoyce, mind giving it one more look? Thanks!

kjellr commented:
Thanks!
